tests/flows/check_upgrade_access.go

```diff
@@ -68,14 +68,11 @@ func CheckUpgradeAccess(network interfaces.Network) {
 	tx, err := exampleMessenger.PauseTeleporterAddress(ownerOpts, teleporterAddress)
 	Expect(err).Should(BeNil())
 	receipt := utils.WaitForTransactionSuccess(ctx, subnetInfo, tx.Hash())
-	blockNum := receipt.BlockNumber
 	pauseTeleporterEvent, err := utils.GetEventFromLogs(receipt.Logs, exampleMessenger.ParseTeleporterAddressPaused)
 	Expect(err).Should(BeNil())
 	Expect(pauseTeleporterEvent.TeleporterAddress).Should(Equal(teleporterAddress))
 
-	// We pass the block number to the call options because the Fuji C-chain endpoint is load balanced,
-	// and between consecutive calls we might be reaching different nodes with different states.
-	isPaused, err = exampleMessenger.IsTeleporterAddressPaused(&bind.CallOpts{BlockNumber: blockNum}, teleporterAddress)
+	isPaused, err = exampleMessenger.IsTeleporterAddressPaused(&bind.CallOpts{}, teleporterAddress)
```

Comment:
What was the issue with this `CallOpts`?

Reply:
There's nothing wrong with specifying it, but if a node handled the request that had not yet seen that specified block height, it would still return a "cannot query unfinalized data" error. That error was more descriptive, but ultimately still results in a failed test.